<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccdcontroller</class>
 <widget class="QMainWindow" name="ccdcontroller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ccdcontroller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QComboBox" name="exptypeComboBox">
       <item>
        <property name="text">
         <string>Exposure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bias</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="testimCheckBox">
       <property name="text">
        <string>Test Image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="exptimeDoubleSpinBox">
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="imtitleLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="imtitleLabel">
       <property name="text">
        <string>Image Title:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="imfilenameLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" rowspan="2">
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="imfilenameLabel">
       <property name="text">
        <string>Image Filename:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QPushButton" name="exposeButton">
       <property name="text">
        <string>Expose</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="exptimeLabel">
       <property name="text">
        <string>Exposure Time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="imtypeLabel">
       <property name="text">
        <string>Image Type:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>imtitleLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>imfilenameLineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
